source/mainnet/net/release-notes/release-notes-lp.rst

```diff
@@ -288,13 +288,23 @@ Wallets
 |bw|
 -------------------------
 
-    April 4, 2024
+    June 20, 2024
 
-    Version 1.5.1 fixes a bug related to the injected script being loaded multiple times which was causing performance issues on the browser.
+    Version 1.5.2 includes several changes and bug fixes.
```

Comment:
@Ivan-Mahda was there a change regarding payload decoding for CIS3-standard?

Reply:
Added update to release notes